contrib/terraform/openstack/variables.tf

```diff
@@ -148,6 +148,12 @@ variable "use_neutron" {
   default     = 1
 }
 
+variable "port_security" {
```

Comment:
How about `port_security_enabled` for clarifying the value should be bool?

Reply:
I don't like long names but yeah, that makes more sense to align with the openstack concept.

Reply:
And done.